Userland/Libraries/LibWeb/MimeSniff/Resource.cpp

```diff
@@ -7,6 +7,256 @@
 #include <LibWeb/Fetch/Infrastructure/URL.h>
 #include <LibWeb/MimeSniff/Resource.h>
 
+namespace {
+
+using namespace Web::MimeSniff;
+using Byte = u8;
+
+class BytePatternTableRow {
+public:
+    BytePatternTableRow(ReadonlyBytes byte_pattern, ReadonlyBytes pattern_mask, Vector<Byte> const& ignored_leading_bytes, StringView mime_type, bool is_tag_terminated = false)
```

Comment:
`Vector<T>&`? In this case, the ignored bytes are static data always right? So why not 

1. A static constexpr data member or
2. ReadonlyBytes?

Reply:
Decided to make the field a `ReadonlyBytes` 👍